config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
   
   get 'server', to: 'server#index'
+  get 'notifications', to: 'notifications#notifications'
```

Comment:
You could change the first controller be called `HomeController`.  I don't know, `notifications#notifications` seems weird.

```ruby
resources :home, only: [:index]
resources :notifications, only: [:index]
```